test/ndt7-testing-backend/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
```

Comment:
Don't do this. Use the real ndt-server libraries and handlers.

Reply:
Yeah, that is indeed the right thing (TM) to do. However, to do that, I need to go back to the server side and make sure I also implement the upload path. This means I am going to put this PR in the backburner for some days, and work instead on the server.